index.html

```diff
@@ -8,7 +8,7 @@ <h2 class="heading">Recent Entries</h2>
   <h1><a href="{{ post.url }}" class="title">{{ post.title }}</a></h1>
   <div class="meta">
     <i class="icon-calendar"></i><time datetime="{{ post.date | date_to_xmlschema }}" pubdate>{{ post.date | date_to_string }}</time>
-    <i class="icon-tags"></i> {{ post | tags }}
+    <i class="icon-tags"></i> {% include tags.html tags=post.tags %}
```

Comment:
This change is reverted here: https://github.com/toshimaru/blog.toshimaru.net/commit/9a01334976d2fab9186e78e27c63fa76be3e7735